scripts/fife_launch_test

```diff
@@ -22,7 +22,7 @@ jobsub_submit \
     --find_setups \
     --export-unquote 'TIMESTAMP%3D%24%28date%20%2B%25Y%25m%25dT%25H%25M%25S%29' \
     --addoutput '*.txt' \
-    --dest '/pnfs/nova/scratch/users/mengel' \
+    --dest '/pnfs/nova/scratch/users/'$USER \
```

Comment:
Is this a typo?  With `'$USER`?  Should it be `$USER'`?

Reply:
No I think that's right; the single quotes would prevent $USER from being expanded, so we just splice it on the end.

Reply:
Ah - I see your point.  Looks good then.